components/todoist/todoist.app.js

```diff
@@ -0,0 +1,112 @@
+const axios = require('axios')
+const querystring = require('querystring')
+const resourceTypes = ['labels','projects','items','notes','sections','filters','reminders','locations','user','live_notifications','collaborators','user_settings','notification_settings'].sort()
```

Comment:
Given that this is static, can we extract it into a JSON file?
You should be able to do something like this:
```js
// resource-types.json
[
  "labels",
  "projects",
  // ...
]

// todoist.app.js
const resourceTypes = require("./resource-types")
```

You can even sort it when creating the JSON file itself, so that the JSON file contains a list of resource types that's already sorted.

Reply:
Pipedream does not support this, but I was able to extract the resource types into a separate .js module.

Reply:
Yeah, I verified and you're right, that's a shame 🙁 
Nice workaround though 🙂